src/renderer/styles/theme.js

```diff
@@ -130,6 +130,31 @@ const animations = {
   fadeInGrowX: props => css`${fadeInGrowX} ${animationLength} ${easings.outQuadratic} forwards`,
 };
 
+const overflow = {
+  x: css`
+    overflow-y: scroll;
+    overflow-x: overlay;
+    &:hover {
+      --track-color: ${p => p.theme.colors.palette.text.shade30};
+    }
+  `,
+  y: css`
+    overflow-y: scroll;
+    overflow-y: overlay;
```

Comment:
why is it defining twice?

Reply:
its a css fallback on scroll if overlay can't be applied :+1: